diff --git a/node_modules/react-native-reanimated/lib/reanimated2/jestUtils.js b/node_modules/react-native-reanimated/lib/reanimated2/jestUtils.js
index d8befd8..638728f 100644
--- a/node_modules/react-native-reanimated/lib/reanimated2/jestUtils.js
+++ b/node_modules/react-native-reanimated/lib/reanimated2/jestUtils.js
@@ -153,15 +153,15 @@ export const setUpTests = (userConfig = {}) => {
         // for Jest in version 28+
         const { expect: expectModule } = require('@jest/globals');
         expect = expectModule;
+        require('setimmediate');
+        frameTime = Math.round(1000 / config.fps);
+        config = Object.assign(Object.assign({}, config), userConfig);
+        expect.extend({
+            toHaveAnimatedStyle(received, expectedStyle, config = {}) {
+                return compareStyle(received, expectedStyle, config);
+            },
+        });
     }
-    require('setimmediate');
-    frameTime = Math.round(1000 / config.fps);
-    config = Object.assign(Object.assign({}, config), userConfig);
-    expect.extend({
-        toHaveAnimatedStyle(received, expectedStyle, config = {}) {
-            return compareStyle(received, expectedStyle, config);
-        },
-    });
 };
 export const getAnimatedStyle = (received) => {
     return getCurrentStyle(received);
